assets/app/views/deployments.html

```diff
@@ -9,14 +9,210 @@
+                      <dt>Change of:</dt>
+                      <dd>Config</dd>
+                    </span>
+                  </span>
+                </div>
+              </dl>
+            </div>
+          
+            <div class="well" style="margin-bottom: 10px;" ng-repeat="deployment in deploymentsByDeploymentConfig[deploymentConfigName] | orderObjectsByDate : true">
+              <h3>
+                {{deployment.metadata.name}}
+                <span ng-if="(deployment | annotation:'deploymentVersion') == deploymentConfig.latestVersion">(latest)</span>
+              </h3>
+              <div>
+                <dt>Created:</dt>
+                <dd><relative-timestamp timestamp="deployment.metadata.creationTimestamp"></relative-timestamp></dd>
+              </div>
+              <div>
+                <span>
+                  <dt>Status:</dt>
+                  <dd>
+                    <span>{{deployment | annotation:'deploymentStatus'}}</span>
+                    <span ng-switch="deployment | annotation:'deploymentStatus'" class="hide-ng-leave">
+                      <span ng-switch-when="Running"> for <duration-until-now timestamp="deployment.metadata.creationTimestamp"></duration-until-now></span>
+                      <span ng-switch-when="New"> for <duration-until-now timestamp="deployment.metadata.creationTimestamp"></duration-until-now></span>
+                      <span ng-switch-when="Pending"> for <duration-until-now timestamp="deployment.metadata.creationTimestamp"></duration-until-now></span>
+                    </span>
+                    <span ng-switch="deployment | annotation:'deploymentStatus'" class="hide-ng-leave">
+                      <span style="margin-left: 5px;" ng-switch-default class="fa fa-refresh fa-spin" aria-hidden="true"></span>
+                      <span ng-switch-when="Complete" class="fa fa-check text-success" aria-hidden="true"></span>
+                      <span ng-switch-when="Failed" class="fa fa-times text-danger" aria-hidden="true"></span>
+                    </span>
+                  </dd>
+                </span>
+              </div>
+              <div>
+                <dt>Labels:</dt>
+                <dd>
+                  <span ng-if="!deployment.metadata.labels">none</span>
+                  <span ng-repeat="(labelKey, labelValue) in deployment.metadata.labels">{{labelKey}}={{labelValue}}<span ng-show="!$last">, </span></span>
+                </dd>
+              </div>
+              <div>
+                <dt>Selectors:</dt>
+                <dd>
+                  <span ng-if="!deployment.spec.selector">none</span>
+                  <span ng-repeat="(labelKey, labelValue) in deployment.spec.selector">{{labelKey}}={{labelValue}}<span ng-show="!$last">, </span></span>
+                </dd>
+              </div>
+              <div>
+                <dt>Replicas:</dt>
+                <dd>{{deployment.spec.replicas}}</dd>
```

Comment:
if `deployment.status.replicas` is not equal to `deployment.spec.replicas`, should surface that somehow...